com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs

```diff
@@ -145,36 +147,19 @@ public bool showAdditionalProperties
             }
         }
 
-        /// <summary>
-        /// Start a scope for additional properties.
-        /// This will handle the highlight of the background when toggled on and off.
-        /// </summary>
-        /// <returns>True if the additional content should be drawn.</returns>
-        protected bool BeginAdditionalPropertiesScope()
```

Comment:
Do we really want to remove that from public namespace?
Anytime a users uses anything else than PropertyFields, the additional properties implementation won't work.

Reply:
I can do the scope protected if you want.

Reply:
I think is not released, that's why I removed. But correct me if I am wrong

Reply:
I don't think it's released yet indeed. But my point is more that users can (and probably will) have UI code without PropertyFields (I know we had quite a few at least for a while) so if we don't expose this anymore, they won't be able to properly implement additional properties.
Or maybe I'm missing something here?

Reply:
I've recovered the functions.